xarray/tests/test_indexing.py

```diff
@@ -6,12 +6,85 @@
 
 from xarray import Dataset, DataArray, Variable
 from xarray.core import indexing, utils
+from xarray.core.pycompat import integer_types
 from . import TestCase, ReturnItem
 
 
+def canonicalize_indexer(key, ndim):
```

Comment:
I was so lazy to update some unit tests that still use these functions.
Yes. I updated them.